ip.go

```diff
@@ -28,8 +28,8 @@ var (
 // following byte sequences is considered a loopback multiaddr.
 var loopbackPrefixes = [][]byte{
 	{ma.P_IP4, 127}, // 127.*
-	IP6LinkLocalLoopback.Bytes(),
```

Comment:
I think this might have been wrong anyway, and made it only match `fe80::1` :) `fe80` is a `/10`, so it should've compared the first 1.25 bytes. The highest possible address in that range starts with `febf`.

Reply:
Oh, `IsIP6LinkLocal()` actually needs that fixed

Reply:
And the comment of `IsIPLookback()` needs an update